test/eth.ens.js

```diff
@@ -103,6 +103,8 @@ function isExpectedReceipt(receipt) {
 }
 
 describe('ens', function () {
+    this.timeout(5000);
```

Comment:
Can you elaborate on why exactly this overall timeout is required? 5s are quite a time :/

Reply:
Thanks for catching. Fixed in f037c31